cardano-api/src/Cardano/Api/LocalStateQuery.hs

```diff
@@ -78,6 +79,12 @@ data LocalStateQueryError
   -- ^ The query does not support Byron addresses.
   deriving (Eq, Show)
 
+-- | UTxO query filtering options.
+data QueryFilter
+  = Address [Address]
```

Comment:
Nitpick: I would call this constructor `FilterByAddress`. Otherwise, at a glance, it seems like a constructor for an actual address.

Reply:
Also, why not stick with a `Set` of `Address`es here? It seems like this list is immediately converted in the code below using `Set.fromList` anyways.